src/commands/Moderation/warn.ts

```diff
@@ -31,6 +31,12 @@ var command: commandInterface = {
                 return false;
             }
 
+            if(args.indexOf(' ') < 0) { // check if there are at least two arguments
+                message.channel.send('Correct usage: `warn [member] [reason]`'); // if not, send the correct usage of the command.
```

Comment:
Could you change this to `Reason needs to be specified`? I was going to change it myself but I can't commit to your repo

Reply:
That wouldn't be really nice because if there are no arguments, it would still prompt `Reason needs to be specified`.

Reply:
That's good though, isn't it? Because it would be a good idea to force people to provide a reason for the warning. Also, if the code prevents the help menu from popping up when just executing `?!warn`, that should be changed. @Jet132 Isn't that preferable or no?

Reply:
If anything, we can wait to implement the requirement of a reason in a later change, but the thing that should still be fixed, if it's currently presented, is that when `?!warn` is used without any argument, it should still provide the help menu for that command.

Reply:
It does provide the help menu when you don't provide any arguments.

Reply:
Oh, right, sorry.
I changed it.